reemgawad/aircams
# Total price of rental

What Changed:

- Date selector changed to range
- Total price is calculated and displayed based on # days selected

How to test: 

- GO TO /cameras, try to rent camera, pick dates. Total price displayed should = daily price * # of days selected: 
<img width="544" alt="Screen Shot 2022-05-19 at 3 26 33 PM" src="https://user-images.githubusercontent.com/100733281/169386813-75ecf1bf-98c4-463e-9679-d332c37d13f3.png">
